src/git.py

```diff
@@ -1,28 +1,44 @@
 import subprocess
 import os
 
-def get_filecontent(filepath: str, filehash: str):
-    # Get rid of `\` on Windows
-    filepath = filepath.replace('\\', '/')
-
+def get_repo_path():
     # Get absolute path to current Git repo
     try:
-        repopath = str(subprocess.check_output('git rev-parse --show-toplevel'))
+        repopath = str(subprocess.check_output(['git', 'rev-parse', '--show-toplevel']))
         # Clean output
         repopath = repopath.lstrip("b'").rstrip("n'").rstrip('\\')
+        return repopath
     except subprocess.CalledProcessError as e:
         print(f'Could not determine the Git repo path. Error: {e.returncode}')
 
+def get_filecontent(filepath: str, filehash: str):
+    # Get rid of `\` on Windows
+    filepath = filepath.replace('\\', '/')
+    repopath = get_repo_path()
     relpath = os.path.relpath(filepath, repopath)
     # Get rid of `\` on Windows
     relpath = relpath.replace('\\', '/')
 
     # Get file content from relpath and filehash
     try:
-        filecontent = str(subprocess.check_output(f'git show {filehash}:{relpath}'))
+        filecontent = str(subprocess.check_output(['git', 'show', f'{filehash}:{relpath}']))
         # Clean output
         filecontent = filecontent.lstrip("b'").rstrip("'").replace('\\n', '')
     except subprocess.CalledProcessError as e:
         print(f'Could not read {filepath} from {filehash}. Error: {e.returncode}')
 
     return filecontent
+
+def get_changed_jsons(path: str, commit_hash: str):
```

Comment:
It doesn't seem like the function only targets JSON files, the name should be more generic. If we change the extension one day, we don't want to change the variable/function names everywhere.